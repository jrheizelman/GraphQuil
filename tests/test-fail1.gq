/* 
 * Author: Gemma Ragozzine
 * Tests failure of conditional statements 
 */

int main () {
	
	int i;
	i = 1;
	while(i) {
		test();
	}	
}

int test(){
	1 + 2;
}