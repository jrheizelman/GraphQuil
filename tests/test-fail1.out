Fatal error: exception Failure("While loop must evaluate on a boolean expression")